src/client/JoinPrivateLobbyModal.ts

```
@@ -71,6 +93,41 @@ export class JoinPrivateLobbyModal extends LitElement {
+                            <div class="participants-list" style="margin-top: 0.5rem;">
+                              ${this.lobbyParticipants.map(
+                                (participant) =>
+                                  html`<span class="participant-tag" style="display: inline-block; margin: 0.2rem; padding: 0.3rem 0.6rem; background-color: #e0e0e0; border-radius: 4px; font-size: 0.9rem; color: #000 !important;">${participant.slice(0, 6)}...${participant.slice(-4)}</span>`,
+                              )}
+                            </div>
+                          `
+                        : ""}
+                    </div>
+                    ${this.onChainJoinError
+                      ? html`
+                          <div style="margin-top: 0.5rem; padding: 0.5rem; background-color: #f8d7da; border: 1px solid #f5c6cb; border-radius: 4px; color: #721c24;">
```

ESLint

[error] This line has a length of 166. Maximum allowed is 120.

(max-len)